src/modules/refresh.js

```diff
@@ -0,0 +1,32 @@
+import message from './message.js';
+
+export default async () => {
+  if (localStorage.getItem('gameid') === null) {
+    message('Please wait for the game ID to be loaded.');
+  } else {
+    // call api to get names/scores list
+    const gameid = localStorage.getItem('gameid');
+    const uri = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameid}/scores/`;
+    try {
+      const response = await fetch(uri, { method: 'GET' });
+      if (response.ok) {
+        const jsonresponse = await response.json();
+        // parse the data into the list
```

Comment:
✅ When the Refresh button is click, it should hit the GET API endpoint causing the scores list to change 👍🏻